app/_hub/kong-inc/prometheus/index.md

```diff
@@ -185,14 +182,18 @@ allow access to the `/metrics` endpoint to Prometheus:
    Prometheus servers use to access the metric data. Access through any
    firewalls would also need to be configured.
 
-2. You can proxy the Admin API through Kong itself then use plugins to limit
+1. If the [Status API](https://docs.konghq.com/latest/configuration/#status_listen)
+   is enabled, then its `/metrics` endpoint can be used.
+   This is the preferred method for nodes configured as a Data Plane.
```

Comment:
This is the preferred method for **ALL** types of installations.

Reply:
@hbagdi just checking; did the first option ever work? using RBAC blocks access, and I couldn't get Prometheus to connect since it does not have an option for key-auth (basic auth + a token one only)?
Did I miss something? or did we document something that doesn't work?

Reply:
You could use a request-transformer plugin to inject the token only for `/metrics` path.
The first option was primarily targeted towards OSS users.

Reply:
that was option 2. Option 1:

> Kong Enterprise users can protect the admin /metrics endpoint with an RBAC user that the Prometheus servers use to access the metric data. Access through any firewalls would also need to be configured.

I couldn't find any way for Prometheus to pass the access token? So this is just incorrect documentation then?